zap/src/main/java/org/parosproxy/paros/control/Control.java

```diff
@@ -382,7 +387,6 @@ private static String wrapEntriesInLiTags(List<String> entries) {
     public void exitAndDeleteSession(String sessionName) {
         shutdown(false);
         Model.getSingleton().getDb().deleteSession(sessionName);
-
```

Comment:
This white space (blank line) change could be reverted, it's not wrong but it's also not really relevant.